### DISCUSSION TITLE
Does GitHub Pages block archive.today?

### DISCUSSION THREAD

**Discussion by notriddle** [community/community#22637] [Other Feature Feedback, Questions, & Ideas]
<p>When I try to archive a page that is hosted on its own domain, <a href="https://archive.is/MHdrB" rel="nofollow noopener">it gives a network error</a>.</p>
<p>When I try to archive a page that uses the <a href="http://github.io" rel="nofollow noopener">github.io</a> domain, <a href="https://archive.is/1LrWo" rel="nofollow noopener">archive automatically uses a proxy to get to it</a>.</p>

**Comment by lee-dohm** [comment, marked as answer]
<p>To my knowledge, GitHub Pages doesn’t block that site specifically. You would have to follow up with the owners of archive.today to get more information on why the errors are occurring.</p>
